README.md

```diff
@@ -25,6 +25,10 @@ clear
 
 **To get started, try out the [codelab](https://g.co/codelabs/clasp)!**
 
+You can also try clasp in Gitpod, a one-click online IDE for GitHub:
+
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/google/clasp/blob/master/demo/)
```

Comment:
@takanakahiko can you please update this PR so that it relocates under `docs/Gitpod`?

- `demo/Code.gs` -> `docs/Gitpod/Code.gs`
- `demo/README.md` -> `docs/Gitpod/README.md`

Then update the link in `/README.md`

Reply:
Sure